core/tee/sub.mk

```diff
@@ -29,15 +29,11 @@ cppflags-tee_svc.c-y += -DTEE_IMPL_VERSION=$(TEE_IMPL_VERSION)
 srcs-y += tee_svc_cryp.c
 srcs-y += tee_svc_storage.c
 
-ifeq (y,$(CFG_RPMB_FS))
-srcs-y += tee_rpmb_fs_common.c
-srcs-y += tee_rpmb_fs.c
-else
-srcs-y += tee_fs_common.c
-endif
+srcs-$(CFG_RPMB_FS) += tee_rpmb_fs.c
+srcs-$(CFG_REE_FS) += tee_ree_fs.c
+ifneq ($(filter y,$(CFG_REE_FS) $(CFG_RPMB_FS)),)
```

Comment:
Is this `ifneq` required as in `tee_svc_storage.c`, it is stated that `At least one filesystem must be enabled`?